apps/docs/pages/guides/database/postgres/setup-replication-external.mdx

```diff
@@ -0,0 +1,60 @@
+import { createDecipheriv } from 'crypto'
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Replicate to another Postgres database using Logical Replication',
+  description:
+    'Example to setup logical replication using publish-subscribe to a Postgres database outside of Supabase',
+  footerHelpType: 'postgres',
+}
+
+For this example, you will need:
+
+- A Supabase project
+- A PostgreSQL database (running v10 or newer)
+
+We will be running commands on both of these databases to publish changes from our Supabase database to our external database.
+
+1. We need to create our `publication` so let's run this on our **Supabase database**:
+
+```sql
+CREATE PUBLICATION example_pub;
+```
+
+2. Also on our **Supabase database**, let's create a `replication slot`:
+
+```sql
+select pg_create_logical_replication_slot('example_slot', 'pgoutput');
+```
+
+3. Now we will connect to our **external database** and subscribe to our `publication` Note: This will need a direct connection to your database and you can find the connection info in the [Dashboard](https://supabase.com/dashboard/project/_/settings/database)):
```

Comment:
```suggestion
3. Connect to the **external database** and subscribe to the `publication` Note: This will need a direct connection to your database. You can find the connection info in the [Dashboard](https://supabase.com/dashboard/project/_/settings/database):
```